integrations/ScriptLoader.js

```diff
@@ -2,17 +2,28 @@
 import logger from "../utils/logUtil";
 
 const defaultAsyncState = true;
+const defaultNonNativeSDKState = "false";
```

Comment:
We don't need these anymore.

Reply:
Addressed.

Reply:
Only removed `const defaultNonNativeSDKState = "false";` this line